tests/test_waku_netconfig.nim

```diff
@@ -0,0 +1,351 @@
+{.used.}
+
+import
+  chronos,
+  libp2p/multiaddress,
+  testutils/unittests
+
+import
+    ../../waku/node/config,
+    ../../waku/waku_enr/capabilities,
+    ../../apps/wakunode2/external_config
+
```

Comment:
Sorry, this is a personal obsession hehe. If possible I recommend always to keep one empty line max.
```suggestion
```

Reply:
Hahaha done, deleted that line and also a couple more instances in the PR with extra empty lines :)